tutorials/large-messages/index.mdx

```diff
@@ -0,0 +1,377 @@
+---
+meta:
+  title: Create a serverless architecture that manage large messages, with Scaleway Messaging and Queuing NATS, Serverless Functions and Object Storage.
+  description: This page show an example of serverless architecture for asynchronous files conversion based on messages, functions, and triggers.
+content:
+  h1: Create a serverless architecture that manage large messages, with Scaleway Messaging and Queuing NATS, Serverless Functions and Object Storage.
+  paragraph: This page show an example of serverless architecture for asynchronous files conversion based on messages, functions, and triggers.
+categories:
+  - messaging
+  - functions
+tags: message nats serverless serverless-functions functions serverless-triggers triggers storage object-storage terraform large-messages
+dates:
+  validation: 2024-01-22
+  posted: 2024-01-22
+---
+
+# Introduction
+
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [set up authentication for the Terraform provider](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs#authentication)
+</Message>
+
+In this tutorial, we will see how to create a simple architecture that stores images and converts them to PDF automatically. 
+The goal is to understand how to transfer large message through a messaging service. 
+To do this, we will use a NATS server available in the Scaleway messaging service. 
+We will also use the Object storage service to store our images and the Serverless Function service to convert them.
+You can retrieve the full source code in [this repository](https://github.com/rouche-q/serverless-examples/tree/main/projects/large-messages/README.md)
+
+## Prerequisites
+
+For this tutorial you will need: 
+
+- Terraform
+- Docker
+- [NATS Cli](https://github.com/nats-io/natscli)
+- [AWS Cli configured with Scaleway credentials](https://www.scaleway.com/en/docs/storage/object/api-cli/object-storage-aws-cli/#how-to-install-the-aws-cli)
+
+## Architecture
+
+<Lightbox src="scaleway-large-messages-architecture.webp" alt="" />
+
+The architecture of this tutorial is very basic. It's structured around 2 elements:
+
+- A producer which is a shell script that upload the image to the Object Storage bucket and send a message to the NATS server.
+- A consumer which is a serverless function in Python that receive the message, parse it,
+  extract the image url to download it from the bucket, convert it to a PDF and reupload it.
```

Comment:
```suggestion
<Lightbox src="scaleway-large-messages-architecture.webp" alt="" />

This tutorial's architecture is very basic, consisting of two main components:

1. A producer implemented as a shell script responsible for uploading an image to the Object Storage bucket and sending a message to the NATS server.
2. A consumer implemented as a serverless Python function designed to receive the message, parse it, extract the image URL, download it from the bucket, convert it to a PDF, and then re-upload the resulting PDF.
```

Reply:
Totally agree with the suggestion, I'd just replace "basic" with "simple"